dev/defaults.yml

```diff
@@ -7,7 +7,7 @@ vm:
   postgresql:
     # enable a separate postgres vm but do not use it unless
     # use-vm-for-external is set.
-    start: false
+    start: true
```

Comment:
Please change to false.

Reply:
Yes - if you haven't already found, you can set this up in dev/config.yml to override the defaults.